ARrehab/ARrehab/Minigames/ExpressionGame/ExpressionViewController.swift

```diff
@@ -194,8 +194,10 @@ extension ExpressionViewController {
         // figure out percentage through current expression
         let timeSinceShown = Date().timeIntervalSince(shownAt)
         // calculate actual points earned (considering precentLeft)
+        // Minimum 1 point, max maxPoints + 1.
         let points = Int(Double(self.maxPointsAwardedPerExpression) * (1.0 - (timeSinceShown / self.timeIntervalPerExpression))) + 1
         
+        // When will points ever be less than 0?
```

Comment:
Think if time ran out just as the user was making the expression it could be possible? Not sure though